src/main/java/racingcar/domain/car/Position.java

```diff
@@ -0,0 +1,57 @@
+package racingcar.domain.car;
+
+import java.util.Objects;
+
+import racingcar.domain.movement.MovementStrategy;
+
+public class Position {
+
+    public static final int DEFAULT_POSITION = 0;
+    public static final int MIN_POSITION = 0;
+
+    private int position;
+
+    public Position() {
+        this(DEFAULT_POSITION);
+    }
+
+    public Position(int position) {
+        validate(position);
+        this.position = position;
+    }
+
+    private void validate(int position) {
+        if (position < MIN_POSITION) {
+            throw new IllegalArgumentException("자동차의 위치는 0이상의 양수 값을 가져야 합니다.");
+        }
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public boolean isSame(int otherPosition) {
+        return position == otherPosition;
+    }
+
+    public void move(MovementStrategy strategy) {
```

Comment:
MovementStrategy가 Position까지 내려올 필요가 있을까요?
```suggestion
    public void move(int distance) {
```